### DISCUSSION TITLE
Unable to distinguish when having duplicate models

### DISCUSSION THREAD

**Discussion by AnotiaWang** [open-webui/open-webui#4495] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** 0.3.12
- **Ollama (if applicable):** 

- **Operating System:** macOS Sequoia Public Beta 3
- **Browser (if applicable):** Vivaldi 6.8.3381.50 (Stable channel) (arm64)

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When there are multiple models with the same name, the UI should clearly distinguish which model comes from which provider.

## Actual Behavior:

User can't distinguish models' providers, and if user select one of them, the UI "selects" all models that have the same name.

## Description

**Bug Summary:**
The UI can't distinguish models that have the same name.

## Reproduction Details

**Steps to Reproduce:**
1. In admin panel, add two connections that have the same model
2. Go to home page
3. Open the models dropdown, and select one of the duplicate models.

## Logs and Screenshots

**Browser Console Logs:**

**Docker Container Logs:**

**Screenshots/Screen Recordings (if applicable):**
<img width="529" alt="image" src="https://github.com/user-attachments/assets/0fa36b43-b488-4f90-9a4c-ff5f1cec2ce7">

## Additional Information

Beside this issue, I have another suggestion: Add the provider's name to the tooltip beside external models, so that we can tell which provider does models belong to.
<img width="521" alt="image" src="https://github.com/user-attachments/assets/0292b266-57e3-4b0b-a0a3-b7be6cd11277">

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
https://openwebui.com/f/hub/openai_proxy_pipe

**Reply by AnotiaWang** [reply]
Thanks
